powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java

```diff
@@ -248,6 +249,55 @@ void activationListForUserTest() throws PowerAuthClientException {
         assertNotEquals(0, listResponse.size());
     }
 
+    @Test
+    void testGetActivationListForUserPagination() throws PowerAuthClientException {
+        // Prepare the base GetActivationListForUserRequest
+        GetActivationListForUserRequest baseRequest = new GetActivationListForUserRequest();
```

Comment:
Let's use the modifier `final` everywhere to ensure we are not reassigning the variable.